src/main/java/springboot/demo/repository/ReviewRepository.java

```diff
@@ -11,4 +13,11 @@
 
     @Query("SELECT r.text FROM Review r")
     List<String> findAllText();
+
+    @Modifying
+    @Transactional
```

Comment:
what does this annotation?(answer in chat)

Reply:
When I used method without annotation I received TransactionRequiredException. https://stackoverflow.com/a/42569935/13229824